src/Parser/EnhancementShaman/CPM_ABILITIES.js

```diff
@@ -0,0 +1,84 @@
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+
+export const SPELL_CATEGORY = {
+  ROTATIONAL: 'Rotational Spell',
+  ROTATIONAL_AOE: 'Spell (AOE)',
+  COOLDOWNS: 'Cooldown',
+  OTHERS: 'Spell',
+};
+
+const CPM_ABILITIES = [
```

Comment:
It would be nice if *all* abilities were here in one form or another. As an example look at the Disc Priest CastEfficiency which has *rotational spell*, *cooldowns*, *spells* and *utility* categories. It even has Shackle Undead so that the list is complete.